### PR TITLE
update api walkthrough doc

### PR DESCRIPTION
**Pull request by czepluch**
This PR improves some language and grammar of the API walkthrough.

It also updates the API calls and answers where it was needed.

The echo node still remains to be fixed before the documentation regarding the echo node can be tested in order to see if it's still valid

### CODE REVIEW

File: docs/api_walkthrough.rst [hunk 1]
```diff
@@ -123,7 +124,7 @@ Here it's interesting to notice that a ``channel_identifier`` has been generated
 
 Depositing to a channel
 ------------------------
-A payment channel is now open between the user's node and a counterparty. However, since only one of the nodes has deposited to the channel, only that node can make transfers at this point in time. Now would be a good time to notify the counterparty that a channel has been opened with it, so that it can also deposit to the channel. All the counterparty needs in order to do this is to use the endpoint consisting of a combination of the ``token_address`` and the ``participant_address``:
+A payment channel is now open between the node of the user and a counterparty. However, since only one of the nodes has deposited to the channel, only that node can make transfers at this point in time. Now would be a good time to notify the counterparty that a channel has been opened with it, so that it can also deposit to the channel. All the counterparty needs in order to do this is to use the endpoint consisting of a combination of the ``token_address`` and the ``participant_address``:
```
Comment (jomuel) [inline docs/api_walkthrough.rst]: I think s-genitive was correct here. Just googled when to use what and it said of-genitive is only for objects ("the nodes of the network", but "the user's node")

File: docs/api_walkthrough.rst [hunk 2]
```diff
@@ -188,68 +189,68 @@ Connecting to an already existing token network is quite simple. All that is nee
...
 
-   {
-       "funds": 2000
-   }
+    {
+        "funds": 2000
+    }
 
-This will automatically connect to and open channels with three random peers in the token network, with 20% of the funds deposited to each channel. Furthermore it will leave 40% of the funds initially unassigned. This will allow new nodes joining the network to open bi-directionally funded payment channels with this node in the same way that it just opened channels with random nodes already in the network. The default behaviour of opening three channels and leaving 40% of the tokens for new nodes to connect with, can be changed by adding ``"initial_channel_target": 3`` and ``"joinable_funds_target": 0.4`` to the payload and adjusting the default value.
+This automatically opens channels with three random peers in the token network. With 20% of the funds deposited to each channel. Furthermore it leaves 40% of the funds initially unassigned. This allows new nodes joining the network to open payment channels with this node in the same way that it just opened channels with random nodes in the network.
```
Comment (jomuel) [inline docs/api_walkthrough.rst]: The ''With ... each channel" part is not a whole sentence, let's leave the comma in front of it.

File: docs/api_walkthrough.rst [hunk 3]
```diff
@@ -188,68 +189,68 @@ Connecting to an already existing token network is quite simple. All that is nee
...
-The user node is now connected to the token network for the AET token, and should have a path to all other nodes that have joined this token network, so that it can transfer tokens to all nodes participating in this network. See the :ref:`Transferring tokens <transferring-tokens>` section for instructions on how to transfer tokens to other nodes.
+The user node is now connected to the token network for the RTT token. It should also have a path to all other nodes that have joined this token network. This means that it can transfer tokens to all nodes participating in this network. See the :ref:`Transferring tokens <transferring-tokens>` section for instructions on how to transfer tokens to other nodes.
 
 
 .. _leave-network:
 
 Leave
 ------
-If at some point it is desired to leave the token network, the ``leave`` endpoint is available. This endpoint will take care of closing and settling all open channels for a specific in the token network:
+If at some point it is desired to leave the token network, the ``leave`` endpoint is available. This endpoint takes care of closing and settling all open channels for a specific  token network:
```
Comment (jomuel) [inline docs/api_walkthrough.rst]: Extra whitespace between 'specific' and 'token'

File: docs/api_walkthrough.rst (hunk 2 above)
Comment (czepluch) [reply docs/api_walkthrough.rst]: You're right. I am trying to use shorter sentences in general, but it doesn't always work out too well. Thanks for pointing it out. Will fix.

**Pull request merged**